--- a/lib/acts_as_xapian/acts_as_xapian.rb
+++ b/lib/acts_as_xapian/acts_as_xapian.rb
@@ -129,15 +129,15 @@ module ActsAsXapian
     end

     # check for a config file
-    config_file = Rails.root.join("config","xapian.yml")
+    config_file = Pathname.new("/var/lib/alaveteli/config/xapian.yml")
     @@config = YAML.load_file(config_file)[environment] if config_file.exist?
     @@config ||= {}

     # figure out where the DBs should go
     if config['base_db_path']
-      db_parent_path = Rails.root.join(config['base_db_path'])
+      db_parent_path = config['base_db_path']
     else
-      db_parent_path = File.join(File.dirname(__FILE__), 'xapiandbs')
+      db_parent_path = Pathname.new('/var/lib/alaveteli/xapiandbs')
     end

     # make the directory for the xapian databases to go in
@@ -877,7 +877,7 @@ module ActsAsXapian
         if verbose
           STDOUT.puts("ActsAsXapian.destroy_and_rebuild_index: Rebuilding #{model_class}")
         end
-        model_class.find_each do |model|
+        model_class.find_each(batch_size: 100) do |model|
           if verbose
             STDOUT.puts("ActsAsXapian.destroy_and_rebuild_index      #{model_class} #{model.id}")
           end
