--- a/app/models/mail_server_log.rb
+++ b/app/models/mail_server_log.rb
@@ -107,14 +107,17 @@ class MailServerLog < ApplicationRecord
       result[queue_id] = [] unless result.key?(queue_id)
       result[queue_id] = (result[queue_id] + emails).uniq
     end
+    # log lines from dovecot yield a nil => [] element in the hash, remove them
+    result.delete(nil)
     result
   end

   # Returns nil if there is no queue id
   def self.extract_postfix_queue_id_from_syslog_line(line)
     # Assume the log file was written using syslog and parse accordingly
-    m = SyslogProtocol.parse("<13>" + line).content.match(/^\S+: (\S+):/)
-    m[1] if m
+    re = Regexp.new(/^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\S{5,13}|[a-zA-Z]{3} .\d \d\d:\d\d:\d\d) \S+ postfix\S+: ([0-9A-F]{10,12}):/)
+    m = re.match(line)
+    m[2] if m
   end

   # We also check the email prefix so that we could, for instance, separately handle a staging and production
