--- a/app/models/info_request.rb
+++ b/app/models/info_request.rb
@@ -1846,8 +1846,9 @@ class InfoRequest < ApplicationRecord
       true
     elsif feature_enabled?(:accept_mail_from_anywhere)
       true
-    elsif user.features.enabled?(:accept_mail_from_poller)
-      source == :poller
+    elsif source == :poller
+      # accept pop email for all users
+      true
     else
       source == :mailin
     end
